### DISCUSSION TITLE
Help building on Windows

### DISCUSSION THREAD

**Discussion by lotusstudio** [fltk-rs/fltk-rs#724] [Q&A]
Dear sir,
    I failed to build my project due to the following error:
------------------------------------
PS D:\coding\Rust\fltk-rs> cargo build
    Blocking waiting for file lock on build directory
   Compiling fltk-sys v1.0.17
   Compiling fltk v1.0.18
error: could not find native static library `cfltk`, perhaps an -L flag is missing?

error: aborting due to previous error

error: could not compile `fltk-sys`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
--------------------------
I'm using Windows 10 OS. Rust version is:
nightly-x86_64-pc-windows-msvc (default)
rustc 1.53.0-nightly (727d10156 2021-04-27)

Please help. Thanks in advance.

**Comment by MoAlyousef** [comment]
Can you try in your Cargo.toml:
```toml
[dependencies]
fltk = { version = "1", features = ["fltk-bundled"] }
```

Also you need to follow the instructions on building for windows, which can be found in the readme, the docs and wiki. Also when opening an issue, please follow the issue template which asks to provide the output of `cargo build -vv`.

**Comment by lotusstudio** [comment]
Problem solved.  In fact , I don't know why this error happened neither how disappeared. It's weird . Due to the network connection? Maybe.

**Reply by cn-hew** [reply]
我也遇到了你这样的问题， 先使用
```
[dependencies]
fltk = { version = "1"}
```
这时候会编译`fltk-sys`，从日志上来看似乎版本是1.2.9, 我改了配置，下面的配置不会报错
```
[dependencies]
fltk-sys="1.2.9"
fltk = { version = "1.2.10",features = ["fltk-bundled"] }
```
